controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -153,9 +154,14 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 		if err := r.reconcileCephClientHealthChecker(); err != nil {
 			return reconcile.Result{}, err
 		}
-
-		if err := r.reconcileNoobaaAccount(); err != nil {
-			return reconcile.Result{}, err
+		//Provider cluster already has a noobaa system and is not required to have a
+		//noobaa account in order to connect to a
+		//remote cluster as opposed to client/hosted clusters
```

Comment:
// A provider cluster already has a NooBaa system and does not require a NooBaa account 
// to connect to a remote cluster, unlike client clusters.
// A NooBaa account only needs to be created if the storage consumer is for a client cluster.

Reply:
updated